producer-c/producer-cloudwatch-integ/KvsProducerSampleCloudwatch.cpp

```diff
@@ -180,26 +210,53 @@ INT32 main(INT32 argc, CHAR *argv[])
             frame.presentationTs = frame.decodingTs;
             frameIndex++;
         }
+        CHK_LOG_ERR(retStatus);
+
+        if (frame.frameData != NULL) {
+            SAFE_MEMFREE(frame.frameData);
+        }
 
+        freeDeviceInfo(&pDeviceInfo);
+        freeStreamInfoProvider(&pStreamInfo);
+        freeKinesisVideoStream(&streamHandle);
+        freeKinesisVideoClient(&clientHandle);
+        freeCallbacksProvider(&pClientCallbacks); // This will also take care of freeing canaryStreamCallbacks
+        DLOGI("Memory allocated:%llu bytes", getInstrumentedTotalAllocationSize());
```

Comment:
Still see the print line - have you uploaded the new commit?

Reply:
Something went wrong :/
Let me git reset and upload the changes again

Reply:
Done